test/integration/test_mpfaMultiDim.py

```diff
@@ -34,9 +34,7 @@ def setup_cart_2d(nx):
     for e, d in gb.edges():
         # Compute normal permeability
         gl, _ = gb.nodes_of_edge(e)
-
-        d_l = gb.node_props(gl)
-        kn = 1.0 / np.mean(d_l[pp.PARAMETERS][kw]["aperture"])
+        kn = 1.0 / aperture
```

Comment:
I prefer defining common variables outside loops, instead of defining inside, and then pick it up later. But that is a matter of taste.

Reply:
I agree. I just didn't bother to make more changes than I had to...

Reply:
And it's not purely a matter of taste. I would argue that defining variables multiple times increases the risk of bugs.